packages/fxa-shared/metrics/glean/web/cadRedirectDesktop.ts

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Did we need to run `yarn glean-generate` here?

Weird this was missed. I added this in content-sever and must have only generated it there instead of in both packages...

Reply:
I was seeing errors locally, but I'll remove for this PR.